connectors.go

```diff
@@ -148,6 +148,16 @@ func (c *Client) UpdateConnectorConfig(name string, config ConnectorConfig) (*Co
 	return connector, response, err
 }
 
+// GetConnectorTaskStatus gets the status of task for a connector.
+//
+// See: https://docs.confluent.io/current/connect/references/restapi.html#get--connectors-(string-name)-tasks-(int-taskid)-status
+func (c *Client) GetConnectorTaskStatus(name string, taskID int) (*TaskState, *http.Response, error) {
```

Comment:
On second thought, could these methods take a `TaskID` as single argument, instead of the weaker constituent basic types?